### PR TITLE
(#5489) Use yaml in e2e install

### PR DESCRIPTION
**Pull request by tdiesler**

### CODE REVIEW

**Review by squakez — APPROVED** [review]

**Review by oscerd — APPROVED** [review]

**Comment by tdiesler** [comment]
This may have failed for unrelated reasons, could we pls give this another shot?

File: e2e/knative/openapi_test.go [hunk 1]
```diff
@@ -33,6 +33,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+/*
+	go test -v -tags=integration ./e2e/knative --run TestOpenAPIService
```
Comment (squakez) [inline e2e/knative/openapi_test.go:37]: Is this comment needed?

**Comment by squakez** [comment]
Let me know if this is good to merge, thanks!

**Comment by tdiesler** [comment]
Ready to merge

**Pull request merged**